src/wallet/rpcwallet.cpp

```diff
@@ -3937,8 +3937,7 @@ UniValue z_getmigrationstatus(const UniValue& params, bool fHelp) {
         throw runtime_error(
             "z_getmigrationstatus\n"
             "Returns information about the status of the Sprout to Sapling migration.\n"
-            "In the result a transactions is defined as finalized if and only if it has\n"
-            "at least ten confirmations.\n"
+            "A transaction is defined as finalized if it has at least ten confirmations.\n"
```

Comment:
```suggestion
            "Note: A transaction is defined as finalized if it has at least ten confirmations.\n"
```

Reply:
I agree with Eirik it is jarring to jump from the definition of the RPC to a comment about "finalized". I think appending with "Note:" is a good compromise.

Reply:
I like adding `Note:` here, but we also have a "note" already on the line below `Note: It is possible that manually created...` If we add `Note:`, I would be nice to change the second `Note: ` to `Also, `.